javalib/src/main/scala-3/java/lang/annotation/Retention.scala

```diff
@@ -1,3 +1,3 @@
 package java.lang.annotation
```

Comment:
Missing explanation? Perhaps the file in particular is self explanatory, I am more sure about this in other files